launch-game.sh

```diff
@@ -1,15 +1,17 @@
 #!/bin/sh
+set -o errexit || exit $?
+
 ENGINEDIR=$(dirname "$0")
-if command -v mono >/dev/null 2>&1 && [ "$(grep -c .NETCoreApp,Version= ${ENGINEDIR}/bin/OpenRA.dll)" = "0" ]; then
+if command -v mono >/dev/null 2>&1 && [ "$(grep -c .NETCoreApp,Version= "${ENGINEDIR}/bin/OpenRA.dll")" = "0" ]; then
 	RUNTIME_LAUNCHER="mono --debug"
 else
 	RUNTIME_LAUNCHER="dotnet"
 fi
 
 if command -v python3 >/dev/null 2>&1; then
-	 LAUNCHPATH=$(python3 -c "import os; print(os.path.realpath('$0'))")
+	 LAUNCHPATH=$(python3 -c 'import os,sys; print(os.path.realpath(sys.argv[1]))' "$0")
```

Comment:
is/was LAUNCHPATH ever correct as it seems to include the basename `launch-game.sh`? or was it meant to be a file and not a dir.

alternative: `LAUNCHPATH=$(dirname $(reallink -f "$0"))` or `LAUNCHPATH=$(reallink -f "$0")`

with no python dependency.

Reply:
bash -x on old script indeed shows `LAUNCHPATH=/home/../OpenRA/launch-game.sh`

Reply:
`LAUNCHPATH` is supposed to contain the `launch-game.sh`. Also, `readlink -f` is a non-portable GNU-ism!